### PR TITLE
Fresh new look for README

### PR DESCRIPTION
**Pull request by FireEmerald**
A fresh new look 🎉

~Not yet done completely~, i'm going to update the gif to include the default banner in action, as it looks good and is one of the core elements. I need also to update the translated german readme...

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,114 +1,105 @@
...
+Thanks for their Open-Source licence to their amazing IDEs and addons like [ReSharper](https://www.jetbrains.com/resharper) for Visual Studio.
 
 ## License: GPLv2
 
+<a href="https://app.fossa.io/projects/git%2Bgithub.com%2FBelphemur%2FSoundSwitch?ref=badge_large" target="_blank"><img alt="FOSSA Status" align="right" src="https://app.fossa.io/api/projects/git%2Bgithub.com%2FBelphemur%2FSoundSwitch.svg?type=large"></a>
+
 Copyright (C) 2015 Jeroen Pelgrims
 
-Copyright (C) 2015-2018 Antoine Aflalo
+Copyright (C) 2015-2019 Antoine Aflalo
```
Comment (Belphemur) [inline README.md]: Since you're at it, let's put 2020 :D

**Comment by Belphemur** [comment]
I like the new look 👍 

Tell me when you're satisfied with it.

Also there are some new translation needed for German on Weblate ;)

**Comment by FireEmerald** [comment]
@Belphemur Ready to be merged, but keep in mind to change [this](https://github.com/Belphemur/SoundSwitch/pull/371/files#diff-04c6e90faac2675aa89e2176d2eec7d8R10) statement to a absolute url.

**Comment by FireEmerald** [comment]
I also included a _dark_ version of the preview image for https://soundswitch.aaflalo.me as the white won't fit in the dark theme.

**Pull request merged**